src/crud.js

```diff
@@ -0,0 +1,12 @@
+class Tasks {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const activities = [];
+
+const newTask = new Tasks('', false, 0);
+activities.push(newTask);
```

Comment:
- [ ] I don't see any array of some simple *to do tasks* (array of objects)  🤔 Kindly, make sure you follow these project  requirements:

- [ ] 1. You should have an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]. Your code might look like this:

   ```javascript
   // ⚠️ Note: Your code doesn't have to be implemented exactly like this, this is only "one" example of how it can be implemented.
   const tasks = [
     {
       description: 'An example description in string',
       completed: false,
       index: 1,
     },
   ];
   ```

- [ ] 2. You should have a function that iterates over the tasks array and populates the HTML list item element for each task. Your code might look like this:

   ```javascript   
   const populateTasks = (task, parentElement) => {
     const { description, completed, index: id } = task;
     const innerHTML = `
            <p>${description}</p>
                  .......`;
     parentElement.appendChild(newTodoListElement);
   };
   ```
- Remember. On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task. You have to make sure the first index is 1 and not zero this way when you create a todo list, the index starts from 1 instead of 0